draft-ietf-rtcweb-jsep.xml

```diff
@@ -4076,6 +4076,65 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
             process, as a remote offer or answer, unmodified SDP coming
             from any other implementation of this specification.</t>
         </section>
+        <section title="Processing RTP packets"
+                 anchor="sec.rtp.demux">
+          <t>When an RTP packet is received by a transport (typically a
+          DtlsTransport), that packet MUST be routed to the correct
+          RtpReceiver. For each transport, the following steps MUST be
```

Comment:
Do we need to make it just a bit more explicit that if there are 2 UDP ports in use, then there are two transports and separate sets of tables for both ? What you have now is totally correct, I just worry that people reading this for first time might miss this is all per transport.  Don't feel strongly about this, if you think it is fine as it, works for me